service/start.go

```diff
@@ -78,4 +92,9 @@ func (dependency *Dependency) Start(service *Service, details dependencyDetails,
 		Ports:      extractPorts(dependency),
 		NetworksID: []string{networkID, sharedNetworkID},
 	})
+	if err != nil {
+		return
+	}
+	err = container.WaitForContainerStatus(namespace, container.RUNNING, time.Minute) //TODO: be careful with timeout
```

Comment:
a minute is really long. If it was in the cli it might be ok but for an internal function I think it's definitely too long

Reply:
What timeout you think is ok?

Otherwise, we could pass the timeout duration as a parameter of the `Start` and `Stop` function.